src/main/resources/templates/store.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="fragments/layout :: head('Assigned orders')">
+    <meta charset="UTF-8">
+    <title>Assigned orders</title>
+</head>
+<body>
+<nav th:replace="fragments/layout :: navbar(~{ :: .mr-auto}, ~{ :: .ml-auto})">
+    <ul class="navbar-nav mr-auto">
+        <li class="nav-item active">
+            <a class="nav-link" th:href="@{/}">Home</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link active" th:href="@{/store}">Store</a>
+        </li>
+    </ul>
+    <ul class="navbar-nav ml-auto">
+        <li class="nav-item">
+            <a class="nav-link" th:href="@{/logout}">Logout</a>
+        </li>
+    </ul>
+</nav>
```

Comment:
Thymeleaf shenanigans, do it like this 

```suggestion
<nav th:replace="fragments/layout :: navbar(~{ :: #navbarRight}, ~{ :: #navbarLeft})">
    <ul class="navbar-nav mr-auto" id="navbarRight">
        <li class="nav-item active">
            <a class="nav-link" th:href="@{/}">Home</a>
        </li>
        <li class="nav-item">
            <a class="nav-link active" th:href="@{/store}">Store</a>
        </li>
    </ul>
    <ul class="navbar-nav ml-auto" id="navbarLeft">
        <li class="nav-item">
            <a class="nav-link" th:href="@{/logout}">Logout</a>
        </li>
    </ul>
</nav>
```